ci/build.ps1

```diff
@@ -0,0 +1,125 @@
+Set-PSDebug -Trace 1
+
+$env:CONFIGURATION -match '^(?<compiler>\w+)_(?<bits>32|64)(?:-(?<option>\w+))?$'
+$compiler = $Matches.compiler
+$compileOption = $Matches.option
+$bits = $Matches.bits
+$cmakeBuildType = 'RelWithDebInfo'
+$depsCmakeVars = @{
+  CMAKE_BUILD_TYPE = $cmakeBuildType;
+}
+$nvimCmakeVars = @{
+  CMAKE_BUILD_TYPE = $cmakeBuildType;
+  BUSTED_OUTPUT_TYPE = 'nvim';
+  GPERF_PRG = 'C:\msys64\usr\bin\gperf.exe';
+}
+
+if ($compiler -eq 'MINGW')
+{
+  if ($bits -eq 32)
+  {
+    $arch = 'i686'
+  }
+  elseif ($bits -eq 64)
+  {
+    $arch = 'x86_64'
+  }
+  if ($compileOption -eq 'gcov')
+  {
+    $cmakeVars['USE_GCOV'] = 'ON'
+    $uploadToCodecov = $true
```

Comment:
I don't see where `$cmakeVars` is ever used.

Reply:
You're right. It should be `$nvimCmakeVars`.